docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Upgrade UWP apps to .NET 6/WinUI3/Windows App SDK
+description: Use the .NET Upgrade Assistant to upgrade an existing UWP app to .NET 6. The .NET Upgrade Assistant is a CLI tool that helps migrating an app from .NET Framework to .NET 6.
+author: adegeo
+ms.date: 05/17/2022
+---
+# Upgrade a UWP App to .NET 6 and WinUI3 with Windows App SDK capabilities using the .NET Upgrade Assistant
+
+The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line tool that can assist with upgrading .NET Framework WPF apps to .NET 6. This article provides:
```

Comment:
nit:
This is your second chance to front load keywords that can be triggered on in search. The "description" metadata above and the first paragraph of the article (this one) heavily influence search results. I would rewrite this to be closer to the description but don't duplicate the description.